src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -2706,17 +2706,25 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
 
             if (varTypeIsLong(simdBaseType))
             {
-                if (simdSize != 64 && !canUseEvexEncoding())
+                if (TARGET_POINTER_SIZE == 4)
                 {
-                    // TODO-XARCH-CQ: We should support long/ulong multiplication
+                    // TODO-XARCH-CQ: 32bit support
```

Comment:
What's blocking 32-bit support? It doesn't look like we're using any `_X64` intrinsics in the fallback logic?

Reply:
Not sure to be honest, that check was pre-existing, I only changed comment